packages/select/src/FilterMultiSelect/FilterMultiSelect.tsx

```diff
@@ -0,0 +1,31 @@
+import { ListBox } from "./components/ListBox"
+import { MenuFooter, MenuLoadingSkeleton } from "./components/MenuLayout"
+import { MultiSelectOption } from "./components/MultiSelectOption"
+import { Root } from "./components/Root"
+import { SearchInput } from "./components/SearchInput/SearchInput"
+import {
+  ClearButton,
+  SelectAllButton,
+} from "./components/SelectionControlButton"
+import {
+  FilterTriggerButton,
+  RemovableFilterTrigger,
+} from "./components/Trigger"
+
+export const FilterMultiSelect = Object.assign(Root, {
+  TriggerButton: FilterTriggerButton,
+  RemovableTrigger: RemovableFilterTrigger,
+  SearchInput,
+  ListBox,
+  Option: MultiSelectOption,
+  SelectAllButton,
+  ClearButton,
+  MenuFooter, // For layout
+  MenuLoadingSkeleton, // Menu Loading Skeleton example
+})
+
+// Here only export component props as we would like to enforce consumer to use dot notation component.
+export * from "./components"
+export * from "./provider"
+export * from "./types"
+export { getSelectedOptionLabels, getSelectedOptionKeys } from "./utils"
```

Comment:
I'm trying to remember if we spoke about this, but it feels weird to exporting from here instead of an index file

Reply:
I might remember it incorrectly but agreed it is more natural to have them in index file so moved!